indico/modules/events/clone.py

```diff
@@ -173,12 +174,6 @@ def _clone_contrib_settings(self, new_event):
             'submitters_can_edit_custom': contribution_settings_data['submitters_can_edit_custom']
         })
 
-    def _clone_subcontrib_settings(self, new_event):
```

Comment:
:bomb: You are still calling `_clone_subcontrib_settings` - please always test code you modified (ie in this case you'd test cloning an event).